contrib/docker/docker-entrypoint.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 if [ "$#" -ne 0 ]; then
     exec "$@"
 elif [ "$SUPERSET_ENV" = "local" ]; then
-    flask run -p 8080 --with-threads --reload --debugger  --host=0.0.0.0
+    flask run -p 8088 --with-threads --reload --debugger  --host=0.0.0.0
```

Comment:
If you change the port number make sure to update the docs that are referencing this